packages/terra-menu/src/_MenuContent.jsx

```diff
@@ -267,20 +270,35 @@ class MenuContent extends React.Component {
           fill={<h1 className={cx('header-title')}>{this.props.title}</h1>}
         />
       </div>
-    ) : <div />;
+    );
+
+    let header = <div />;
+
+    if (this.props.index > 0) {
+      header = subMenuHeader;
```

Comment:
This [chunk of code](https://github.com/cerner/terra-framework/pull/977/files#diff-8821ddb6189a1accf05f7aef26338373R255-R272) can directly move in here.

Reply:
updated here dcae9eedc320f50ade9e1b88b8d942181019b385